core/primitives/src/lib.rs

```diff
@@ -286,3 +286,25 @@ impl From<LogLevel> for log::Level {
 		}
 	}
 }
+
+/// Converts the given value into an `u64` by following the Substrate calling convention.
+///
+/// When Substrate calls into Wasm it expects a fixed signature for functions exported
+/// from the Wasm blob. The return value of this signature is always a `u64` in little endian.
```

Comment:
I am not sure what is meant here since `endianess` can only apply to values stored in memory. When we are talking about just regular values the values are always stored in direct order.